### ISSUE TITLE
Makers to check if the txfee is sufficient before signing the transaction

### ISSUE THREAD

**Issue by raedah** [JoinMarket-Org/joinmarket#424]
Have them send an !error so the taker can go into the recover process without continuing to wait.

**Comment by chris-belcher** [comment]
Yep, completely agreed. I think I wrote this down somewhere months ago.

**Comment by AdamISZ** [comment]
Yeah, I think we're all in agreement, discussed this on IRC. As long as our takers can recover correctly, it's a good improvement. And yes, this has doubtless been suggested in another Issue, but let's face it, none of us knows where that is :)

**Comment by chris-belcher** [comment]
Probably in this one https://github.com/JoinMarket-Org/joinmarket/issues/58
